ci/publish-github-release.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# publish-github-release.sh creates a "release" in the github repo and associates with the build tag
+
+# usage publish-github-release.sh REPO VERSION
+set -o errexit
+set -o pipefail
+
+if [[ -z "${PULUMI_BOT_GITHUB_API_TOKEN:-}" ]]; then
+    echo "error: PULUMI_BOT_GITHUB_API_TOKEN must be set"
+    exit 1
+fi
+
+if [ "$#" -ne 2 ]; then
+    >&2 echo "usage: $0 REPO VERSION"
+    exit 1
+fi
+
+PAYLOAD="{\"tag_name\": ${2}, \"name\": ${2}}"
```

Comment:
The `${2}` value likely needs quoting to be valid JSON - quotes from the shell invocation will be removed.

Reply:
ah good point here!